docker/dalc-docker-compose.yml

```
@@ -3,20 +3,26 @@ version: '3'
 services:
   dalc0:
     container_name: dalc0
-    image: "ghcr.io/celestiaorg/dalc:old-stable"
+    image: "ghcr.io/celestiaorg/dalc:sha-b46432b"
     environment:
+      - NODE_TYPE=light
       - MY_VAL_ADDRESS=celes1dm7fp7uwrr5y68v48gg8jhd2cku7vt7f4l8w0s
+      - CELESTIA_PRIVATE_GENESIS=54A8B66D2BEF13850D67C8D474E196BD7485FE5A79989E31B17169371B0A9C96
     command: [
-            "dalc", "start",
-            "--home", "/root"
+            "celestia", "light",
+            "--node.store", "/dalc",
+            "start",
+            "--headers.trusted-peers",
+            "/ip4/192.167.1.0/tcp/2121/p2p/12D3KooWL8z3KARAYJcmExhDsGwKbjChKeGaJpFPENyADdxmEHzw",
```

Why is this one `192.167.1.0`? Is there some list of which IPs do what?
```suggestion
            "/ip4/192.167.10.0/tcp/2121/p2p/12D3KooWL8z3KARAYJcmExhDsGwKbjChKeGaJpFPENyADdxmEHzw",
```

---

There's no holistic list in one place. The `-docker-compose.yml` files have them
```
networks:
      localnet:
        ipv4_address: 192.167.3.0 
 ```
 the rough pattern is they all start with `192.167`. 
 - Core nodes(s)are on `192.167.10.x`
 - Bridge node(s) are on `192.167.1.x`
 - Light node(s) are on `192.167.2.x`
 - DALC is on `192.167.3.x`
 - Ethermint is on `192.167.4.x`